lib/src/sync/window.dart

```diff
@@ -28,11 +28,11 @@ abstract class Windows {
 /// Upon use, the window will automatically be set as active.
 abstract class Window {
   /// The size of the window.
-  @Deprecated('JSON wire legacy support, emulated for newer browsers')
+  @Deprecated('JSON wire legacy support, unsupported for newer browsers')
```

Comment:
What does unsupported here means? if it truly doesn't work, why cant we just remove this?